packages/vanilla/src/sass/components/accordion/_index.scss

```diff
@@ -1,148 +1,127 @@
-//@use '../../base/colors' as *;
-@use '../../typography' as *;
-@use './states' as *;
-//@use '../../tokens/font' as *;
-
-.cbp-accordion {
-  margin: 24px 0;
+.cbp-accordion-group {
+  margin-top: var(--cbp-space-6x);
+  margin-bottom: var(--cbp-space-6x);
   width: 100%;
+}
 
-  .cbp-accordion__item {
-    margin-bottom: 12px;
-  
-    &:last-of-type {
-      margin-bottom: 0;
-    }
-  }
-
-  .cbp-accordion__title {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    width: 100%;
-    color: var(--cbp-color-text-darkest);
-    background: var(--cbp-color-interactive-neutral-lighter);
-    border: none;
-    outline: transparent;
-    padding: 16px;
-    transition: all 300ms;
-
-    div:first-child i {
-      color: var(--cbp-color-text-darker);
-      font-size: var(--cbp-font-size-heading-md);
-      transition: transform .2s ease-in-out;
-    }
+.cbp-accordion__item:not(:last-child) {
+  margin-bottom: var(--cbp-space-3x);
```

Comment:
Did you test if this is needed? Margins should collapse, so this should basically disappear for the last item since the wrapper has a larger bottom margin.

Reply:
Yes, this should add bottom margins to every accordion item except the last accordion item otherwise there would need to be another line with `cbp-accordion__item:last-child { margin-bottom: 0}` so its should make the css shorter.

Reply:
This isn't true. I'm saying you don't need to specify no margin for the last item because the wrapper has a margin and margins collapse - only the larger (wrapper's) margin will be applied. If you don't omit the last item, there's no change. I verified in-browser.